cmd/nethealth/main.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/gravitational/satellite/lib/nethealth"
+
+	"github.com/gravitational/trace"
+	"github.com/gravitational/version"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func init() {
+	version.Init("v0.0.1-master+$Format:%h$")
+}
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatal(trace.DebugReport(err))
```

Comment:
`log.WithError(err).Error("Failed.")` Otherwise the stack trace will be unreadable due to quoting.